Tests/eppo/UniversalFlagConfigTests.swift

```diff
@@ -56,7 +92,7 @@ final class UniversalFlagConfigTest: XCTestCase {
                 return
             }
             XCTAssertEqual(thrownError.errorCode, 101, "Error code should be 101 indicating a JSON parsing issue")
-            XCTAssertEqual(thrownError.localizedDescription, "Data corrupted: The given data was not valid JSON.")
+            XCTAssertTrue(thrownError.localizedDescription.hasPrefix("Data corrupted: The given data was not valid JSON."))
```

Comment:
this error message got longer now with the `underlyingError` added to it; this is sufficient